CHANGELOG.md

```diff
@@ -121,6 +121,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed `CombinedLoader` and `max_size_cycle` didn't receive a `DistributedSampler` ([#10374](https://github.com/PyTorchLightning/pytorch-lightning/issues/10374))
 
 
+- Fixed the call to `extract_batch_size` only when its required ([#10408](https://github.com/PyTorchLightning/pytorch-lightning/pull/10408))
```

Comment:
The changelog is not clear what the issue was that got fixed. IMO it should always mention the problem. Describing the solution is optional.

Reply:
```suggestion
- Fixed an issue to not extract batch_size when batch_size inside `LightningModule.log` is set by the user ([#10408](https://github.com/PyTorchLightning/pytorch-lightning/pull/10408))
```

Reply:
better?

Reply:
Sorry, I think it is worse 😅 

I think it is either

> Fixed an issue that prevented Lightning to extract the batch size even though it was set by the user in LightningModule.log


or

> Fixed an issue that caused Lightning to extract the batch size even though it was set by the user in LightningModule.log


Totally not sure what you are trying to fix. Given the description of the PR, I probably think it is the latter.